README.md

```diff
@@ -1,7 +1,162 @@
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About</a>
+      <ul>
+        <li><a href="#built-with">Tools used</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgements">Acknowledgements</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+VisualMQ is a supportive application for IBM MQ systems. Created in Unity, VisualMQ is designed to create 3D representations of MQ systems. For this purpose, we have created novel 3D models representing the different components of MQ. VisualMQ is aimed at being a novel tool that revolutionizes the way end-users obtain an understanding of the systems functioning.  As opposed to the current inspection methods for MQ systems,  the relationship between different components is emphasized and data is represented in such a way to make it as easy as possible to understand. 
+
+
+### Built With
+
+This section should list any major frameworks that you built your project using. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+* [Unity](https://unity.com/)
+* [Blender](https://www.blender.org/)
+* ...
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+VisualMQ is available under the ... LICENSE. 
+
+There are a multitude of ways to use the product. The first is cloning this repository and importing it into Unity version ????. From there, it is possible to build the project manually and play around with it. Other options involve downloading available [binaries](https://github.com/VisualMQ/VisualMQ/releases) for popular systems or the available [web-demo](https://visualmq.diallom.com/demo/).
+
+### Prerequisites
+
+Usage of VisualMQ requires a valid set of queue manager details. For this valid IBM Cloud details are required. For this we recommend observing the [official tutorial](https://www.ibm.com/cloud/garage/dte/tutorial/tutorial-mq-ibm-cloud) the first two steps "Creating MQ Services On IBM Cloud" and "Creating A Queue Manager On IBM Cloud" to create one or more queue managers. For each of these queue managers the following credentials will be necessary:
+- API KEY
+- Username
+- Host URL
+- Queue Manager Name
+
+The tips under the tutorial's section "Administering MQ Using MQ Console" give partial instruction on how to retrieve these. 
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+TBA Screenshots and video
+
+<!-- ROADMAP -->
```

Comment:
Report is not part of the read-me (and far too long to be expecting people to read it just to know what is planned). Potential features should be listed within the read-me as short and quick glance of what is/can come.

Reply:
I only expect the next group of UCL students who will continue to work on the project to read the report thoroughly. but I think it's too detailed to put it into README (it's not in the READMEs you referenced below neither). and since we already know that there will be a new group of UCL students working on this, and John is the product owner who is responsible for the roadmap, I wouldn't put it here.

Reply:
Okay! To remove